demos/src/Marks/TextStyle/React/AddGlobalAttributes.ts

```diff
@@ -0,0 +1,70 @@
+import '@tiptap/extension-text-style'
+
+import { Extension, Mark } from '@tiptap/core'
+
+export const AddGlobalAttributes = Extension.create({
+  name: 'color',
+
+  addOptions() {
+    return {
+      types: ['textStyle'],
+    }
+  },
+
+  addGlobalAttributes() {
+    return [
+      {
+        types: this.options.types,
+        attributes: {
+          color: {
+            default: null,
+            parseHTML: element => element.style.color?.replace(/['"]+/g, ''),
+            renderHTML: attributes => {
+              if (!attributes.color) {
+                return {}
+              }
+
+              return {
+                style: `color: ${attributes.color}`,
+              }
+            },
+          },
+          fontFamily: {
+            default: null,
+            parseHTML: element => element.style.fontFamily,
+            renderHTML: attributes => {
+              if (!attributes.fontFamily) {
+                return {}
+              }
+              return {
+                style: `font-family: ${attributes.fontFamily}`,
+              }
+            },
+          },
+          fontSize: {
+            default: null,
+            parseHTML: element => element.style.fontSize?.replace(/['"]+/g, ''),
+            renderHTML: attributes => {
+              if (!attributes.fontSize) {
+                return {}
+              }
+
+              return {
+                style: `font-size: ${attributes.fontSize}`,
+              }
+            },
+          },
+        },
+      },
+    ]
+  },
+})
+
+export const Strong = Mark.create({
```

Comment:
Do we need this for the demo? Can't we use the default Bold extension from `@tiptap/extension-bold`?